package.json

```diff
@@ -30,6 +30,7 @@
   },
   "types": "./types/fh-sync.d.ts",
   "scripts": {
+    "example": "cd examples/basic-express-example/ && npm i && node server.js",
```

Comment:
Personally - I do not like it. We can introduce this in example application package.json